src/components/WebhooksTabContent/WebhooksTabContent.tsx

```diff
@@ -17,6 +17,7 @@ export const WebhooksTabContent = ({ webhooks }: WebhooksTabContentProps) => {
           <DataTable
             columns={webhookColumns}
             data={webhooks}
+            initialSorting={[{ id: 'url', desc: false }]}
```

Comment:
Please make a constant for that:
```ts
const initialSorting = [{ id: 'url', desc: false }]
```

And then:
```suggestion
            initialSorting={initialSorting}
```

Reply:
done